README.md

```diff
@@ -286,6 +149,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] Great work on the project. Please make sure the MIT link in your readme is working properly and redirecting to the MIT license.

![image](https://github.com/Young-Gaius/vet-clinic/assets/62290545/4cbf79b4-fb1c-48d3-957d-6420190a458b)